chall02/dbelinsk.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
+import sys
+
+def encode(c):
+	if c is 'a':
+		return ".-"
+	elif c is 'b':
+		return "-..."
+	elif c is 'c':
+		return "-.-."
+	elif c is 'd':
+		return "-.."
+	elif c is 'e':
+		return "."
+	elif c is 'f':
+		return "..-."
+	elif c is 'g':
+		return "--."
+	elif c is 'h':
+		return "...."
+	elif c is 'i':
+		return ".."
+	elif c is 'j':
+		return ".---"
+	elif c is 'k':
+		return "-.-"
+	elif c is 'l':
+		return ".-.."
+	elif c is 'm':
+		return "--"
+	elif c is 'n':
+		return "-."
+	elif c is 'o':
+		return "---"
+	elif c is 'p':
+		return ".--."
+	elif c is 'q':
+		return "--.-"
+	elif c is 'r':
+		return ".-."
+	elif c is 's':
+		return "..."
+	elif c is 't':
+		return "-"
+	elif c is 'u':
+		return "..-"
+	elif c is 'v':
+		return "...-"
+	elif c is 'w':
+		return ".--"
+	elif c is 'x':
+		return "-..-"
+	elif c is 'y':
+		return "-.--"
+	elif c is 'z':
+		return "--.."
+	elif c is ' ':
+		return " "
```

Comment:
Remember that you are using python, you could have used a dictionary to store the correspondances of each character.  
[Here](https://www.w3schools.com/python/python_dictionaries.asp) is a little link to learn more the basic about python dicts.